executor/infoschema_reader.go

```diff
@@ -1831,18 +1831,27 @@ func (e *memtableRetriever) setNewTiKVRegionStatusCol(region *helper.RegionInfo,
 		} else {
 			row[6].SetInt64(0)
 		}
+		if table.IsPartition {
+			row[9].SetInt64(1)
+			row[10].SetInt64(table.Partition.ID)
```

Comment:
How about the partition table that has multiple partitions? Such as 2 partitions. Maybe we can refer to the partitions table: https://docs.pingcap.com/tidb/stable/information-schema-partitions. And the code is here: https://github.com/pingcap/tidb/blob/master/executor/infoschema_reader.go#L1087

Reply:
@zimulala "How about the partition table that has multiple partitions? Such as 2 partitions. ".
Is it mean "SUBPARTITION_NAME"? 
There is already  "PARTITON_ID" and "PARTITION_NAME " to distinguish between partitions.